judge/admin/contest.py

```diff
@@ -148,7 +156,28 @@ def make_private(self, request, queryset):
 
     def get_urls(self):
         return [url(r'^rate/all/$', self.rate_all_view, name='judge_contest_rate_all'),
-                url(r'^(\d+)/rate/$', self.rate_view, name='judge_contest_rate')] + super(ContestAdmin, self).get_urls()
+                url(r'^(\d+)/rate/$', self.rate_view, name='judge_contest_rate'),
+                url(r'^(\d+)/judge/(\d+)/$', self.rejudge_view, name='judge_contest_rejudge')] + super(ContestAdmin, self).get_urls()
+
+    def rejudge_view(self, request, contest_id, problem_id):
+        if not request.user.has_perm('judge.rejudge_submission'):
+            self.message_user(request, ugettext('You do not have the permission to rejudge submissions.'),
+                              level=messages.ERROR)
+            return
+
+        queryset = ContestSubmission.objects.filter(problem_id=problem_id).select_related('submission')
+        if queryset.count() > 10 and not request.user.has_perm('judge.rejudge_submission_lot'):
```

Comment:
This constant of 10 also appears in https://github.com/DMOJ/site/blob/2e416d7a7ef2100372cd3a467d080dce45e0705d/judge/admin/submission.py#L133

Since we'd be hardcoding it twice, it'd be best to move it into `local_settings.py` (e.g. `DMOJ_REJUDGE_SUBMISSION_LIMIT = 10`) alongside a short comment describing what it does.